playerctl/playerctl-cli.c

```diff
@@ -16,279 +16,359 @@
  * along with playerctl If not, see <http://www.gnu.org/licenses/>.
  *
  * Copyright © 2014, Tony Crisci
+ * Copyright © 2016, Jente Hidskes <hjdskes@gmail.com>
```

Comment:
It doesn't matter because of the OSS license, but my understanding is that your fork constitutes a [derivative work](https://en.wikipedia.org/wiki/Derivative_work) which is not substantially different enough to be protected under copyright. So by that reasoning, there is only one copyright which is my original one.

There are two practical issues here which are 1) contact info for authors and 2) recognition for work. However, they are not actually very important at all because these are much more easily discovered through github rather than the source files. If you want to add a CONTRIBUTORS file go ahead.